root/usr/share/nethesis/NethServer/Template/OpenVpnTunnels/Clients/Modify.php

```diff
@@ -36,6 +36,7 @@
 $remote = $view->fieldset()->setAttribute('template', $T('Remote_label'))
      ->insert($view->textArea('RemoteHost', $view::LABEL_ABOVE)->setAttribute('dimensions', '5x30'))
      ->insert($view->textInput('RemotePort'))
+     ->insert($view->textArea('RemoteNetworks', $view::LABEL_ABOVE)->setAttribute('dimensions', '5x30'))
```

Comment:
I think this is a bit misleading: the user usually do not need to fill the field.
Move it under the `Advanced` tab, just like inside Cockpit

Reply:
I lost my skill in nethgui :)

Reply:
nethgui is obsoleted in 7.9